templates/zerver/api/get-presence.md

```diff
@@ -23,23 +23,7 @@
 
 #### Return values
 
-* `presence`: An object containing the presence details for every type
-  of client the user has ever logged into.
-    * `{client_name}` or `aggregated`: the keys for these objects are
-      the names of the different clients where this user is logged in,
-      like `website`, `ZulipDesktop`, `ZulipTerminal`, or
-      `ZulipMobile`. There is also an `aggregated` key, which matches
-      the contents of the object that has been updated most
-      recently. For most applications, you'll just want to look at the
-      `aggregated` key.
-        * `timestamp`: when this update was received; if the timestamp
-          is more than a few minutes in the past, the user is offline.
-        * `status`: either `active` or `idle`: whether the user had
-          recently interacted with Zulip at the time in the timestamp
-          (this distinguishes orange vs. green dots in the Zulip web
-          UI; orange/idle means we don't know whether the user is
-          actually at their computer or just left the Zulip app open
-          on their desktop).
+{generate_return_values_table|zulip.yaml|/users/{email}/presence:get}
```

Comment:
I think the rendered docs for this are a little broken; we're missing the `aggregated` field (which I think can be the only one we focus on documenting, as we're planning to remove the rest).

Reply:
Solved edited the markdown extension for this case.